backend/config.py

```diff
@@ -2,11 +2,11 @@
 from loguru import logger
 
 
-def load_config(file_path: str = './configs/local/config.yaml.'):
+
+def load_config(file_path: str = '/code/configs/local/config.yaml'):
```

Comment:
Getting this error:
`FileNotFoundError: [Errno 2] No such file or directory: '/code/configs/local/config.yaml'`

Reply:
ah, we should change the cp command to go to here instead of the root config directory. I have a local config dir under the shroud to prevent committing local data

Reply:
Ahhh, that makes sense. I'll make the next commit and keep going